docs/source/user/notebook.rst

```diff
@@ -151,6 +151,10 @@ and select “New Console for Notebook”:
      <iframe src="https://www.youtube-nocookie.com/embed/eq1l7DBngQQ?rel=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
   </div>
 
+.. _kernel_history:
+
+You can iterate through the kernel history in a document cell using ``Alt Up-Arrow`` and ``Alt Down-Arrow``.
```

Comment:
Can this feature be disabled? Should it be opt-in? I am specifically concerned about sending a lot of history which is possibly originating from root_dir other than current; some changes I suggest in other comments (only requesting on first user action triggering `notebook:access-previous-history-entry`/`notebook:access-next-history-entry`, reducing _n_) would alleviate the issue but it still feels not a great thing to do without warning users about it.

Reply:
Yeah, I think that could be a good idea. Could you point me to an example of another feature that is opt-in that I could use as an example?

Reply:
In notebook context, side-by-side rendering, in completer there is documentation panel (though to be honest these are not very well documented).